forge-gui/res/cardsfolder/upcoming/arna_kennerud_skycaptain.txt

```diff
@@ -0,0 +1,13 @@
+Name:Arna Kennerüd, Skycaptain
+ManaCost:2 W U B
+Types:Legendary Creature Human Knight
+PT:4/4
+K:Flying
+K:Lifelink
+K:Ward:Discard<1/Card>
+T:Mode$ Attacks | ValidCard$ Creature.modified+YouCtrl | TriggerZones$ Battlefield | Execute$ TrigDoubleCounters | TriggerDescription$ Whenever a modified creature you control attacks, double the number of each kind of counter on it. Then for each nontoken permanent attached to it, create a token that's a copy of that permanent attached to that creature.
+SVar:TrigDoubleCounters:DB$ MultiplyCounter | Defined$ TriggeredAttackerLKICopy | SubAbility$ DBCopyPermanents
+SVar:DBCopyPermanents:DB$ CopyPermanent | Defined$ Valid Permanent.nonToken+Attached | NumCopies$ 1 | AttachedTo$ TriggeredAttackerLKICopy
```

Comment:
the valid looks incomplete?

Reply:
I've given this a lot of testing and I'm not sure how to approach the valid differently. I see sometimes the copying isn't functioning right, but not sure of the fix

Reply:
Well Attached property alone has no additional context, so it just assumes you're checking against the trait host

check e.g. _Rhuk, Hexgold Nabber_

Reply:
Okay, thank you for the tip! I reworked the `Defined$` parameter to use the `AttachedTo` tag, it seems to work with all valid creatures in testing. I'm still not sure whether the `TriggeredAttackerLKICopy` is correct per the oracle text, but `TriggeredCardLKICopy` didn't work in its place